src/legacy/core_plugins/metrics/common/model_options.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export const MODEL_TYPES = {
+  UNWEIGHTED: 'simple',
+  WEIGHTED_EXPONENTIAL: 'ewma',
+  WEIGHTED_EXPONENTIAL_DOUBLE: 'holt',
+  WEIGHTED_EXPONENTIAL_TRIPLE: 'holt_winters',
+  WEIGHTED_LINEAR: 'linear',
+};
+
+export const MODEL_SCRIPTS = {
```

Comment:
Not sure if it should be in a common folder. Please move it to server folder

Reply:
Moved